components/logins/src/db.rs

```diff
@@ -797,8 +838,19 @@ impl Store for LoginDb {
         Ok(CollectionRequest::new("passwords").full().newer_than(since))
     }
 
-    fn reset(&self) -> result::Result<(), failure::Error> {
-        LoginDb::reset(self)?;
+    fn get_sync_assoc(&self) -> result::Result<StoreSyncAssociation, failure::Error> {
+        let global = self.get_meta(schema::GLOBAL_SYNCID_META_KEY)?;
+        let coll = self.get_meta(schema::COLLECTION_SYNCID_META_KEY)?;
+        Ok(match (global, coll) {
```

Comment:
Nit: I think you can make this an `if let`, if you like, but it's up to you.

Reply:
yes, thanks, that's better.